bundles/core/src/test/java/com/adobe/cq/commerce/core/components/internal/models/v1/product/ProductImplTest.java

```diff
@@ -453,6 +453,28 @@ public void testBundleProductImpl(boolean hasStagedData) throws IOException {
         }
     }
 
+    @Test
+    public void testGiftCardProduct() throws IOException {
+        testBundleProductImpl(false);
+    }
+
+    public void testGiftCardProductImpl(boolean hasStagedData) throws IOException {
```

Comment:
Please rename the parameter to something more meaningful for the test.

Reply:
Just following the same pattern as for the other tests

Reply:
This is about `hasStagedData` which seems not related to gift cards and is probably a left over of copy&paste.

Reply:
Ok, I removed it completely